Nodejs/Tests/Core.UI/BasicIntellisense.cs

```diff
@@ -45,6 +45,7 @@ public sealed class BasicIntellisense : NodejsProjectTest {
 
                 server.MoveCaret(2, 13);
                 solution.ExecuteCommand("Edit.CompleteWord");
+                Keyboard.Type("\t");
```

Comment:
why is this necessary? Edit.CompleteWord should be sufficient, and we should try to fix the ExecuteCommand call if that's not working.